internal/commands/secrets/create_inputs_test.go

```diff
@@ -0,0 +1,102 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+	"github.com/Netflix/go-expect"
+)
+
+func TestCreateInputs(t *testing.T) {
+	for _, tc := range []struct {
+		description    string
+		createInputs   createInputs
+		prepareProfile func(p *cli.Profile)
```

Comment:
is `prepareProfile` even used in this test block?  might be able to get rid of it